src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,165 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int count;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(tail, value, null);
+        if (tail != null) {
+            tail.next = newNode;
+        }
+        tail = newNode;
+        if (head == null) {
+            head = newNode;
+        }
+        count++;
     }
 
     @Override
     public void add(T value, int index) {
```

Comment:
one important question here - does your implementation allow `null` values?

my personal suggestion is to forbid them, since if you have null elements - it may be hard to implement method like `contains(element)` or `remove(element)`

**the same should be considered within `add(value)` method**